doc/book/src/config.md

```diff
@@ -7,12 +7,6 @@ variables, or configuration files.
 Each configuration value has a type, which is either string, number, or
 boolean.
 
-Imagine a configuration value named `config-value` in a program called
-`maelstrom-prog`. This configuration value can be specified via:
-  - The `--config-value` command-line option.
-  - The `MAELSTROM_PROG_CONFIG_VALUE` environment variable.
-  - The `config-value` key in a configuration file.
-
```

Comment:
I found this superfluous at first and a little confusing due to the introduction of the fictitious `maelstrom-prog`, but on second thought maybe it's nice to have a little section at the top here that pretty concisely explains what is explained in detail below.

I'd try to find a way to do it without using `maelstrom-prog` though.

Reply:
It seems like you're enumerating two options:

1. Just cut this whole paragraph and list because it's covered below. Or maybe shorten is to a sentence and mention that the details will be covered below.
2. Use a different example, just not a fictitious program.

You seem to settle on 2. Would you still recommend that?